src/main/java/frc/robot/swerve/ModuleIOReal.java

```diff
@@ -157,7 +145,7 @@ public SwerveModulePosition getModulePosition() {
     @Override
     public void updateOffset(double offset) {
         angleMotor.setPosition(
-                ((encoder.getAbsolutePosition() - offset) * 2048) / SwerveConstants.ANGLE_REDUCTION);
+                ((encoder.getAbsolutePosition() - offset) * SwerveConstants.FALCON_TICKS) / SwerveConstants.ANGLE_REDUCTION);
```

Comment:
It did happen at initialization but when the DIO has a lot of ports being used it fails to update the offset because it takes too much time and than the swerve doesn't work properly. We tried a bunch of solutions and that's the one we ended up with.

Reply:
I see. Have you tried sleeping a bit after configuring these properties, or configuring it in in a for loop a few times to increase consistency?  
Anyway, please open an issue on this in [the WPILib beta repo](https://github.com/wpilibsuite/2024Beta/issues), this does sound like a bug.

Reply:
Also, remember that there aren't ticks anymore